vignettes/Workflow.Rmd

```diff
@@ -0,0 +1,479 @@
+---
+title: "Some Basic Analysis of ChIP-Seq Data"
+authors:
+  - name: "Deepayan Sarkar"
+  - name: "Robert Gentleman"
+  - name: "Michael Lawrence"
+  - name: "Zizhen Yao"
```

Comment:
The PDF doesn't list any authors so it isn't clear who are the authors. Please remove them. It's okay to leave yourself as the converter.